### ISSUE TITLE
[FYI] Improved(?) Random Selector

### ISSUE THREAD

**Issue by derornos** [zzorba/ArkhamCards#441]
Arkhamfiles KOR Team provides:
- random investigator(s) selector with tier (community preference).
- random weakness card(s) selector with ban system.

https://arkhamfiles.github.io/utility.html

1) random investigator(s) selector
The site not only randomly selects investigators based on collections, but also changes the probability of investigators appearing based on community-suggested tier ratings. This feature has satisfied both challenging players and novice players. (maybe limited to 'random hangers' XD)

2)  random weakness card(s) selector with ban system.
In the Korean community, the following weakness card selection method is popular through the site.
- A player draws 3 random weakness cards for own investigator, and the other players look at that investigator's deck and vote for the weakness card that they predict will be the easiest for that investigator.
- That card is banned and that player chooses 1 of the remaining 2 cards for own investigator.
- In addition to that, players can also ban certain Weakness cards. (ex. Doomed, ...)

**Comment by zzorba** [comment]
Yeah... I want to do both of these things.  I will see what can be done about them.

**Comment by derornos** [comment]
I'm glad for your answer. thanks! : )
